### DISCUSSION TITLE
Error al agregar Eluna engine al compilar

### DISCUSSION THREAD

**Discussion by jhoancito** [TrinityCore/TrinityCore#27264] [Q&A] [answered]
hola tengo varios errore ni archivos no encontrado sale en el error como lo soluciono

**Comment by Aokromes** [comment, marked as answer]
this is not eluna support.

**Reply by jhoancito** [reply]
> this is not eluna support.

It is not supported by source and the source does not catch it, the eluna must have an error, the source or how do I put a lua file for my server
